### ISSUE TITLE
Show more tasks

### ISSUE THREAD

**Issue by zeke** [ruyadorno/ntl#11]
Maybe I'm weird, but many of my projects have more than eight scripts. I know I can arrow down through them all, but it would be nice if `ntl` showed all of the scripts.

<img width="616" alt="screen shot 2017-02-23 at 9 49 22 am" src="https://cloud.githubusercontent.com/assets/2289/23271747/8a1f46b2-f9ad-11e6-9a27-3dcf7fc4f376.png">

**Comment by mischah** [comment]
Hej @ruyadorno,

would you accept a PR for that? 
That’s something I am missing in a lot of cases.

Question: Should it default to to show 7 and have an option to show more?

Like:

```
  -s --size   Sets the amount of tasks to show in the list without scrolling
```

**Comment by mischah** [comment]
I’m going to check how Inquirers `pageSize` property reacts when passing a number creater than available choices …

**Comment by ruyadorno** [comment]
hi @mischah yeah def! I'd say for the default value we keep whatever is the default from https://github.com/SBoudrias/Inquirer.js

keep in mind though that I'm hard at work on **iPipeTo** which I meant to reuse here as a way of having a unified codebase as I mentioned in #12 - btw just yesterday I finished a first version of a proper API (https://github.com/ruyadorno/ipt/pull/26) 🎉 

just wanted to let you know that any effort here might end up being removed later on during the migration/rewrite and it might be better to implement it directly on https://github.com/ruyadorno/ipt instead

all that said, please feel free to give it a shot! all help is appreciated ❤️

**Comment by mischah** [comment]
> just wanted to let you know that any effort here might end up being removed later on during the migration/rewrite

No problem, as long as the PR lands in a release 😘 

> it might be better to implement it directly on https://github.com/ruyadorno/ipt instead

To be honest ipt looks to complex for a short dive in. I want to work on a Yeoman feature after finishing this PR. So I prefer to add the feature over here for now.

Cheers, Michael :octocat:

**Comment by ruyadorno** [comment]
AWESOME! 🎆 

Please go ahead, I can cut a minor release as soon as it lands 😉 

thanks!!!

**Comment by ruyadorno** [comment]
thanks @zeke for reporting this one! 🏆 

🎉 landed on release [v1.3.0](https://github.com/ruyadorno/ntl/releases/tag/v1.3.0)
